plugins/digitalocean/personal_access_token.go

```diff
@@ -32,6 +34,33 @@ func PersonalAccessToken() schema.CredentialType {
 		DefaultProvisioner: provision.EnvVars(map[string]sdk.FieldName{
 			"DIGITALOCEAN_ACCESS_TOKEN": fieldname.Token,
 		}),
-		Importer: importer.TryAllEnvVars(fieldname.Token, "DIGITALOCEAN_ACCESS_TOKEN"),
+		Importer: importer.TryAll(
+			importer.TryAllEnvVars(fieldname.Token, "DIGITALOCEAN_ACCESS_TOKEN"),
+			TryDigitalOceanConfigFile(),
+		),
 	}
 }
+
+func TryDigitalOceanConfigFile() sdk.Importer {
+	return importer.TryFile("~/Library/Application Support/doctl/config.yaml", func(ctx context.Context, contents importer.FileContents, in sdk.ImportInput, out *sdk.ImportAttempt) {
```

Comment:
Is a loading configuration from file only available for MacOS? Otherwise, we may want to add support for Linux as well (only these two OS's are supported on the CLI side)

Reply:
I'd say let's wait for the merging of this: https://github.com/1Password/shell-plugins/pull/127

Reply:
I'd also be okay with going ahead and merging this PR. Nothing will break in the mean time on Linux. We can add Linux support in a follow-up PR.

Reply:
I'd opt for the following signature:
```go
importer.MacOnly(
	TryDigitalOceanConfigFile("~/Library/Application Support/doctl/config.yaml"),
),
```